app/src/main/java/com/asfoundation/wallet/home/HomeViewModel.kt

```diff
@@ -163,6 +165,7 @@ constructor(
   val hasSavedEmail = mutableStateOf(hasWalletEmailPreferencesData())
   val isEmailError = mutableStateOf(false)
   val emailErrorText = mutableStateOf(0)
+  private val alreadyGetImpression = mutableStateOf(false)
```

Comment:
if you use the "by", you don't need the use the "value". maybe change to:

`private val alreadyGetImpression by mutableStateOf(false)`